readthedocs/basic/signing-in.rst

```diff
@@ -94,16 +94,21 @@ Signing In as a Bot Account
 ===========================
 
 You can also use Telethon for your bots (normal bot accounts, not users).
-You will still need an API ID and hash, but the process is very similar:
+You will still need an API ID and hash that could be retrived from *any application*
+created before.
 
+So, the process is very similar:
 
 .. code-block:: python
 
     import asyncio
     from telethon import TelegramClient
 
-    api_id = 12345
+    # Use the values retrieved from an application created in my.telegram.org
```

Comment:
```suggestion
    # Use the values retrieved from my.telegram.org
```

I still find "an application" confusing. As long as the text before describes how to create the application there, I don't think it's necessary to repeat the mentions to "application" here.